nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/users/ui/user-listing/user-table/user-table.component.html

```diff
@@ -79,24 +79,37 @@
                     <th mat-header-cell *matHeaderCellDef></th>
                     <td mat-cell *matCellDef="let item">
                         <div class="flex items-center justify-end gap-x-2">
-                            @if (canEditOrDelete(currentUser, item)) {
-                                <div
-                                    class="pointer fa fa-cog primary-color"
-                                    title="Edit"
-                                    (click)="editClicked(item, $event)"></div>
-                            }
-                            @if (canEditOrDelete(currentUser, item)) {
-                                <div
-                                    class="pointer fa fa-trash primary-color"
-                                    title="Remove"
-                                    (click)="deleteClicked(item)"></div>
-                            }
-                            @if (hasAccessPolicies(item)) {
-                                <div
-                                    class="pointer fa fa-key primary-color"
-                                    title="View User Policies"
-                                    (click)="viewAccessPoliciesClicked(item, $event)"></div>
+                            @if (
+                                canEditOrDelete(currentUser, item) || hasAccessPolicies(item) || hasAccessPolicies(item)
+                            ) {
+                                <button
+                                    mat-icon-button
+                                    type="button"
+                                    [matMenuTriggerFor]="actionMenu"
+                                    class="h-16 w-16 flex items-center justify-center icon global-menu">
+                                    <i class="fa fa-ellipsis-v"></i>
+                                </button>
                             }
+                            <mat-menu #actionMenu="matMenu" xPosition="before">
```

Comment:
This menu doesn't close when the edit or user policies dialogs pop up in front:
<img width="1307" alt="Screenshot 2024-05-07 at 5 02 17 PM" src="https://github.com/apache/nifi/assets/713866/70056aec-c345-4b8e-bcda-e4bfb8abeca7">

It does hide if the delete confirmation dialog opens though.

Reply:
https://github.com/apache/nifi/pull/8766#discussion_r1594323472